simulator/risc_v/t/unit_test.cpp

```diff
@@ -196,6 +211,21 @@ TEST_CASE ( "RISCV slo64")
     }
 }
 
+TEST_CASE ( "RISCV RV64 sloi")
+{
+    std::vector<TestData<uint64>> cases = {
+        TestData<uint64>( 0x1F, 0, 0x1F),
+        TestData<uint64>( 0x17, 2, 0x5F),
+        TestData<uint32>( all_ones<uint32>(), 5, bitmask<uint64>(37)),
```

Comment:
Still, it is not a valid C++ code. You need to [build](https://github.com/MIPT-ILab/mipt-mips#build) and [run](https://github.com/MIPT-ILab/mipt-mips#run). Please tell if you have issues with these flows.

Reply:
I have some problems with version cmake and run project. Now I try to solve this problems.

Reply:
What is the problem?

Reply:
My cmake version=3.15.4. When I run "cmake /home/ravil/mipt-v/mips/mipt-mips/simulator -G "Ninja"", I get an error.
BEGIN:
CMake Error at /usr/local/share/cmake-3.15/Modules/CMakeDetermineSystem.cmake:173 (file):
  file failed to open for writing (No such file or directory):

    /build/mipt-mips/CMakeFiles/CMakeOutput.log
Call Stack (most recent call first):
  CMakeLists.txt:10 (project)
END:
-- Configuring incomplete, errors occurred!
CMake Error: Cannot open file for write: /build/mipt-mips/CMakeCache.txt.tmp
CMake Error: : System Error: Permission denied
CMake Error: Unable to open cache file for save. /build/mipt-mips/CMakeCache.txt
CMake Error: : System Error: Permission denied

Reply:
You don't have permissions to write to `/build/`. I suggest creating a directory in `~/`. Alternatively, you may use `sudo`.

Reply:
thank you very much

Reply:
You are welcome.
I've asked whether you had any issue 9 hours ago: https://github.com/MIPT-ILab/mipt-mips/pull/1396#discussion_r519685789.
Next time, please reply to our questions, they are here to help you.